authcontroller.py

```diff
@@ -497,8 +500,16 @@ def invite_organisation():
                     flash(
                         "Organisation Onboarded Successfully!!! Email Communication has been sent to Admin",
                         "success")
+    elif request.method == 'GET':
+        orgInfoData = OrgInfo.select()
```

Comment:
this could be done much more elegantly with list /dict comprehensions.

and even more simple directly dumping query results into JSON:
```
json.dumps(list(OrgInfo.select(OrgInfo.name, OrgInfo.email).dicts()))
```
but since it gets to passed to the form. The form is responsible for rendering data into JSON. For that Jinja has a special filter that deals with all fuss with encoding: "json"
so form would have to get ```list(OrgInfo.select(OrgInfo.name, OrgInfo.email).dicts())``` (I reckon even "list" could be omitted)